creatures/change_hp.lua

```diff
@@ -1,23 +1,25 @@
---= Creatures MOB-Engine (cme) =--
--- Copyright (c) 2015-2016 BlockMen <blockmen2015@gmail.com>
---
--- change_hp.lua
---
--- This software is provided 'as-is', without any express or implied warranty. In no
--- event will the authors be held liable for any damages arising from the use of
--- this software.
---
--- Permission is granted to anyone to use this software for any purpose, including
--- commercial applications, and to alter it and redistribute it freely, subject to the
--- following restrictions:
---
--- 1. The origin of this software must not be misrepresented; you must not
--- claim that you wrote the original software. If you use this software in a
--- product, an acknowledgment in the product documentation is required.
--- 2. Altered source versions must be plainly marked as such, and must not
--- be misrepresented as being the original software.
--- 3. This notice may not be removed or altered from any source distribution.
---
+--[[
+	= Creatures MOB-Engine (cme) =
```

Comment:
If the official name is **Creatures** and this is an **Mob-Engine**, then the `Creatures Mob-Engine` name is correct. We can not suppress the **Creatures** term because this is definitely the _name_ of Engine. If you want to change the designation for Mob-API, there are no problems.
`Creatures Mob-Engine` or `Creatures Mob-API` but not `Mob-API` or `Mob-API (CME)`.
I support the deletion of the CME abbreviation , as this is the official name.